prac_06/programming_language.py

```diff
@@ -0,0 +1,18 @@
+
+class ProgrammingLanguage:
+    """Represents a Programming Language."""
+
+    def __init__(self, name, typing, reflection, year):
+        """Define a programming language with inputted values."""
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def __str__(self):
+        """Return a String of the programming language."""
+        return f"{self.name}, {self.typing} Typing, Reflection={self.reflection}, First appeared in {self.year}"
+
+    def is_dynamic(self):
```

Comment:
This does not work as expected. We don't need a method to simply return an attribute.
We needed a Boolean-returning method to return whether the string variable was "Dynamic".